program/rust/src/accounts.rs

```diff
@@ -105,6 +105,11 @@ pub trait PythAccount: Pod {
     /// have. `INITIAL_SIZE` <= `minimum_size()`
     const MINIMUM_SIZE: usize = size_of::<Self>();
 
+    /// Size of the account data on creation. Usually this is the same as `MINIMUM_SIZE` but it's
+    /// different for `PermissionAccount` because we've added new fields to it. In this case
+    /// we cannot increase `MINIMUM_SIZE` because that would break reading the account.
+    const NEW_ACCOUNT_SPACE: usize = Self::MINIMUM_SIZE;
```

Comment:
If we increase `MINIMUM_SIZE`, that will break all instructions that read from permission account because the actual size of the permission account will be smaller than `MINIMUM_SIZE`. So that would require us to call an instruction that grows the permission account immediately after the upgrade, and I'm trying to avoid that. We can change that if it's easier to do with a migration instruction.